src/lib/attestation/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './usdc'
```

Comment:
.js suffixes; I'd appreciate a lint rule for those

Reply:
Adding this in #24 , will require rebasing and adjusting all imports to use `.ts` extension (going in the direction of modern node support for stripping TS types)